common/core/web/input-processor/src/text/inputProcessor.ts

```diff
@@ -58,12 +58,8 @@ namespace com.keyman.text {
      * @returns     {Object}             A RuleBehavior object describing the cumulative effects of
      *                                   all matched keyboard rules.
      */
-    processKeyEvent(keyEvent: KeyEvent): RuleBehavior {
+    processKeyEvent(keyEvent: KeyEvent, outputTarget: OutputTarget): RuleBehavior {
```

Comment:
Please add the new parameter to the doc comment above (and fix the existing param name `e` in that comment while you are at it 😉)